### DISCUSSION TITLE
How to use svg both as regular images and as svgr?

### DISCUSSION THREAD

**Discussion by dartess** [vercel/turborepo#8698] [Help] [answered]
### Summary

Now I use svgr as a webpack loader for files with the extension `.svg?svgr`. Files without the get parameter are processed as regular images. Is something like this possible with a turbopack? All my attempts did not lead to the fact that turbopack can even process SVGs as images.

**Comment by clement-escolano** [comment, marked as answer]
Hello :-)
There is already some discussion (although with no result yet) over here: https://github.com/vercel/next.js/discussions/65360

**Comment by anthonyshew** [comment]
Hi, thanks for the question! This is the repository for Turborepo, so please direct your Turbopack questions over to the Next.js repository for now.

Looks like @clement-escolano already provided a great pointer. Thanks!
